src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,103 @@
+package duke;
+
+/**
+ * DukeException specifies the message to print when an exception is thrown in Duke.
+ *
+ * @author Gabriel Goh
+ */
+public class DukeException extends Exception {
+
+    public DukeException(String message) {
+        super(message);
+    }
+
+}
+
+class TaskIndexNotInteger extends DukeException {
```

Comment:
Maybe you can include more descriptive header comments for the classes and public methods here? I think there are also other methods in the rest of the code for which header comments can be added.

Reply:
Ok